examples/spouse/input/articles.tsv.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+cd "$(dirname "$0")"
+
+cat /dfs/scratch0/thodrek/signalmedia/signalmedia-1m.jsonl |
```

Comment:
Right.  We should change this to grab from the same dir (`input/`) and put instructions to download the full corpus (people have to go through license agreement and get an email with download link btw).

We'll grab a subset of articles, get NLP markups and include the sentences.tsv.bz2 instead for running our integration tests.